core/domain/config_domain.py

```diff
@@ -496,46 +499,6 @@ def get_all_config_property_names(cls) -> List[str]:
     'rich text svgs.',
     2)
 
-MAX_NUMBER_OF_TAGS_ASSIGNED_TO_BLOG_POST = ConfigProperty(
-    'max_number_of_tags_assigned_to_blog_post',
-    POSITIVE_INT_SCHEMA,
-    'The maximum number of tags that can be selected to categorize the blog'
-    ' post',
-    10
-)
-
 CONTRIBUTOR_DASHBOARD_IS_ENABLED = ConfigProperty(
```

Comment:
This seems to be left here, but really you can delete it and make it True always. (OK if you're planning to do that in a separate PR.)